integration/kubernetes/helm-generate.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+function printUsage {
+  echo "Usage: MODE [UFS]"
+  echo
+  echo "MODE is one of:"
+  echo -e " single-ufs        \t Generate Alluxio YAML templates for a single-master environment using UFS journal."
```

Comment:
nit: rename command and directory name to 
- `singleMaster-ufsJournal`
- `multiMaster-embeddedJournal`

Reply:
I'll change the directory names to the more informative ones you suggested. They can be easily auto-completed. 

But I feel the commands need a lot of typing so I'd prefer to leave the command unchanged as they convey sufficient information what they are doing. The README.md will explain single -> singleMaster and ufs -> ufsJournal (sorry doc will be ready in the next commit).

Reply:
sounds good